iroh-rpc-client/src/client.rs

```diff
@@ -180,140 +188,3 @@ impl Client {
         }
     }
 }
-
-// TODO: write tests for mem transport
-#[cfg(all(test, feature = "grpc"))]
-mod tests {
```

Comment:
this seems like a nice test, why through it all out?

Reply:
This is a test for the status stuff, which is from tonic-health. It basically does not exist in this PR, so there is nothing to test.

Reply:
fair